docs/general/installation/container.md

```diff
@@ -73,6 +73,12 @@ docker run -d \
  jellyfin/jellyfin
 ```
 
+:::note
+
+Replace `uid:gid` if you want to run jellyfin as a specific user/group (must exist in the host `/etc/passwd` file). Exclude the `--user` argument entirely if you want to use the default user.
```

Comment:
I don't think `must exist in the host `/etc/passwd` file` is correct. You can input whatever you want.